tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/OrganisationControllerTests.cs

```diff
@@ -146,12 +236,35 @@ public async Task GetIdByAsync_OrganisationExists_ReturnsTheOrganisation()
             result.Should().BeOfType<OkObjectResult>();
             var objectResult = result as OkObjectResult;
 
-            objectResult.Value.Should().BeEquivalentTo(new OrganisationViewModel
+            objectResult.Value.Should().BeEquivalentTo(organisation,
+                conf => conf.Excluding(c => c.Id).Excluding(c => c.Address).Excluding(c => c.LastUpdated));
+        }
+
+        [Test]
+        public async Task GetByIdAsync_OrganisationAddressIsNull_ReturnsOrganisationWithNullAddress()
+        {
+            var organisation = new Organisation
             {
-                OrganisationId = organisation.Id,
-                Name = organisation.Name,
-                OdsCode = organisation.OdsCode
-            });
+                Id = Guid.NewGuid(),
+                Name = "org name",
+                OdsCode = "ods-code",
+                PrimaryRoleId = "ID 1",
+                CatalogueAgreementSigned = true,
+                Address = null
+            };
+
+            using var controller = OrganisationControllerBuilder
+                .Create()
+                .WithGetOrganisation(organisation)
+                .Build();
+
+            var result = await controller.GetByIdAsync(organisation.Id);
+
+            result.Should().BeOfType<OkObjectResult>();
+            var objectResult = result as OkObjectResult;
+
+            objectResult.Value.Should().BeEquivalentTo(organisation,
```

Comment:
Again here please check the excluding of the address property. The assert should ensure the address is null seen as we are setting the `Address `property to `null`.